### PR TITLE
[Bug] Invalid command format messages

### PR DESCRIPTION
**Pull request by timjkong**
Fix invalid command format message bugs.
Standardise invalid command format messages for all commands.

### CODE REVIEW

File: src/main/java/seedu/address/logic/command/add/AddAccommodationCommand.java [hunk 1]
```diff
@@ -12,24 +12,30 @@
...
+            + PREFIX_END + WanderlustDate.FORMAT;
+
+    public static final String MESSAGE_EXAMPLE = "Example: "
+            + AddCommand.COMMAND_WORD + COMMAND_SEPARATOR + COMMAND_WORD + " "
+            + PREFIX_NAME + "Intercontinental "
             + PREFIX_LOCATION + "80 Middle Road "
             + PREFIX_COST + "250 "
             + PREFIX_START + "2020-09-15 "
-            + PREFIX_END + "30-09-2020 ";
+            + PREFIX_END + "2020-09-30";
```
Comment (lyeyixian) [inline src/main/java/seedu/address/logic/command/add/AddAccommodationCommand.java:36]: The dates need to change according to Jia Wei's new implementations.

File: src/main/java/seedu/address/logic/command/add/AddActivityCommand.java [hunk 2]
```diff
@@ -6,33 +6,35 @@
...
+    public static final String MESSAGE_EXAMPLE = "Example: "
+            + AddCommand.COMMAND_WORD + COMMAND_SEPARATOR + COMMAND_WORD + " "
             + PREFIX_NAME + "Universal Studios Singapore "
             + PREFIX_IMPORTANCE + "5 "
             + PREFIX_LOCATION + "Sentosa "
             + PREFIX_COST + "88 "
-            + PREFIX_DATETIME + "2020-09-16 "
-            + PREFIX_TAG + "fun "
-            + PREFIX_TAG + "theme park";
+            + PREFIX_DATETIME + "2020-09-16 ";
```
Comment (lyeyixian) [inline src/main/java/seedu/address/logic/command/add/AddActivityCommand.java:35]: Same as above.

File: src/main/java/seedu/address/logic/command/add/AddFriendCommand.java [hunk 3]
```diff
@@ -12,18 +12,23 @@
...
-            + PREFIX_PASSPORT + "PASSPORT_NUMBER "
-            + "Example: " + COMMAND_WORD + " "
+            + PREFIX_PASSPORT + "PASSPORT_NUMBER ";
+
+    public static final String MESSAGE_EXAMPLE = "Example: "
+            + AddCommand.COMMAND_WORD + COMMAND_SEPARATOR + COMMAND_WORD + " "
             + PREFIX_NAME + "John Doe "
             + PREFIX_MOBILE + "91234567 "
-            + PREFIX_PASSPORT + "p12345678 ";
+            + PREFIX_PASSPORT + "p12345678";
```
Comment (lyeyixian) [inline src/main/java/seedu/address/logic/command/add/AddFriendCommand.java:29]: Passport number need to change to [STFG] + 7 numbers + [A-Z]

File: src/main/java/seedu/address/logic/command/add/AddTravelPlanCommand.java [hunk 4]
```diff
@@ -9,21 +9,26 @@
...
-            + PREFIX_START + "15-09-2020 "
-            + PREFIX_END + "30-09-2020 ";
+            + PREFIX_START + WanderlustDate.FORMAT + " "
+            + PREFIX_END + WanderlustDate.FORMAT;
+
+    public static final String MESSAGE_EXAMPLE = "Example: "
+            + AddCommand.COMMAND_WORD + COMMAND_SEPARATOR + COMMAND_WORD + " "
+            + PREFIX_NAME + " France "
+            + PREFIX_START + "2020-09-15 "
+            + PREFIX_END + "2020-09-30 ";
```
Comment (lyeyixian) [inline src/main/java/seedu/address/logic/command/add/AddTravelPlanCommand.java:29]: Date issue

**Review by lyeyixian — APPROVED** [review]
LGTM. Just the Mobile, Passport and Date thing need updates

File: src/main/java/seedu/address/logic/command/add/AddAccommodationCommand.java (hunk 1 above)
Comment (jiaweiteo) [reply src/main/java/seedu/address/logic/command/add/AddAccommodationCommand.java:36]: Yup, just put the example date to any date on 2021 to be safe, cause now you can only create a travelplan that is after today's date.

File: src/main/java/seedu/address/logic/command/edit/EditActivityCommand.java [hunk 5]
```diff
@@ -29,20 +29,24 @@
...
+            + PREFIX_COST + "COST "
+            + PREFIX_DATETIME + WanderlustDateTime.FORMAT;
+
+    public static final String MESSAGE_EXAMPLE = "Example: "
+            + EditCommand.COMMAND_WORD + COMMAND_SEPARATOR + COMMAND_WORD + " 1 "
+            + PREFIX_NAME + "Universal Studios Singapore "
+            + PREFIX_IMPORTANCE + "5 "
+            + PREFIX_LOCATION + "Sentosa "
+            + PREFIX_COST + "88 "
+            + PREFIX_DATETIME + "2020-09-16 ";
```
Comment (jiaweiteo) [inline src/main/java/seedu/address/logic/command/edit/EditActivityCommand.java:47]: Date issue

**Review by jiaweiteo — APPROVED** [review]
LGTM, just the date issue

**Pull request merged**